Sources/UI/UITableView+Diff.swift

```diff
@@ -1,4 +1,5 @@
 import UIKit
+import OptiDiff
```

Comment:
To make the linter happy:
```suggestion
import OptiDiff
import UIKit
```

Reply:
I see you did the change already thank you. 💪